homeassistant/data_entry_flow.py

```diff
@@ -321,9 +321,13 @@ async def async_configure(
 
             # If the result has changed from last result, fire event to update
             # the frontend.
```

Comment:
We could explain better what "result has changed" means, maybe:
```suggestion
            # If the result has changed from last result, fire event to update
            # the frontend. The result is considered to have changed if:
            # - The step has changed
            # - The step is same but result type is SHOW_PROGRESS and progress_action
            #   or description_placeholders has changed
```